lib/wicked/wlan.pm

```diff
@@ -69,25 +103,44 @@ sub netns_output {
     return script_output($cmd, @args);
 }
 
+sub netns_run {
+    my ($self, $cmd, @args) = @_;
+    $cmd = 'ip netns exec ' . $self->netns_name . ' ' . $cmd;
+    my $ret = script_run($cmd, @args);
+    die("Timeout on script_run($cmd)") unless defined($ret);
+    return $ret;
```

Comment:
i wonder if eval is more appropriate for this case. wdyt?

Reply:
Don't understand, how would your eval look like?